layouts/default.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div id="app" class="app-viewport">
+    <header class="main-header">
+      <img src="~/assets/images/marianne-logo.svg" class="main-header__logo" alt="République française" />
+      <img src="~/assets/images/pix-logo.svg" class="main-header__logo" alt="Pix" />
```

Comment:
Je me demande si les `alt` qu'on a ajouté ne polluent pas plus qu'autre chose l'expérience des screen readers 🤔 

```suggestion
      <img src="~/assets/images/marianne-logo.svg" class="main-header__logo" alt="" />
      <img src="~/assets/images/pix-logo.svg" class="main-header__logo" alt="" />
```

Reply:
Done